doc/source/operations/rabbitmq.rst

```diff
@@ -0,0 +1,90 @@
+===============================
+Rolling out changes to RabbitMQ
+===============================
+
+Prior to upgrading, some changes need to be rolled out for RabbitMQ.
+This should make RabbitMQ more stable during the upgrade(s). These changes
+are:
+
+* Enable the high availability setting via Kolla-Ansible
+  ``om_enable_rabbitmq_high_availability``.
+* Update RabbitMQ to version 3.9.22
+
+**NOTE:** There is guaranteed to be downtime during this procedure, as it
+requires restarting RabbitMQ and all the OpenStack services that use it. The
+state of RabbitMQ may also need to be reset.
+
+Instructions
+------------
+
+If you are upgrading from Wallaby to Xena, you will need to enable the HA
+config option in
+``/home/ubuntu/kayobe/config/src/kolla-ansible/ansible/group_vars/all.yml``.
```

Comment:
This is the Kolla Ansible source code which should not be modified (and is an absolute path which won't always be valid). Instead, add the variable in `etc/kayobe/kolla/globals.yml`.